README.md

```diff
@@ -9,12 +9,11 @@
 
 <h4 align="center">
   <a href="https://docs.medusajs.com">Documentation</a> |
-  <a href="https://demo.medusajs.com/">Medusa Admin Demo</a> |
   <a href="https://www.medusajs.com">Website</a>
 </h4>
 
 <p align="center">
-An open source composable commerce engine built for developers.
+  Building blocks for digital blocks
```

Comment:
Is it intended to be digital blocks? Sounds weird

Reply:
Haha no - nice catch! Thanks :)